tim_sort/Python/tim_sort.py

```diff
@@ -0,0 +1,13 @@
+
+def tim_sort(arr):  # Function that performs the tim sort
+   arr.sort()
```

Comment:
E111 indentation is not a multiple of four

*Origin: PycodestyleBear (E111), Section: `all.autopep8`.*